server/config/options.go

```diff
@@ -102,10 +104,11 @@ var DefaultConfig = Config{
 	},
 	Auth: AuthConfig{
 		IssuerURL:                "https://tigrisdata-dev.us.auth0.com/",
-		Audience:                 "https://tigris-db-dev-preview-test",
+		Audience:                 "https://tigris-db-api",
 		JWKSCacheTimeout:         5 * time.Minute,
 		LogOnly:                  true,
 		EnableNamespaceIsolation: false,
+		AdminNamespaces:          set.New("tigris-data"),
```

Comment:
We should have a generic name for admin namespace?

Reply:
renamed.